dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java

```diff
@@ -472,6 +475,13 @@ private String getUnsanitizedDisplayValue() {
     }
 
     private String getFieldDisplayValue(String value, DatasetFieldType fieldType) {
+        
+        if (fieldType.getFieldType().equals(FieldType.PERIODO)) {
+            return  "<a href=\"" + this.fieldValue + "\" target=\"_blank\">" + this.fieldValue +
+                    "</a><br>" + PeriodoDictionary.getByUrl(this.fieldValue).get()
+                            .getDetails("<br>");
+        }
```

Comment:
Most likely we need some `isNotBlank` check.

Reply:
Field value cannot be blanc AFAIK. If the teztfield on gui is blanc hie field is not created at all.
There is PeriodoValidator to validate field value before saving.

Reply:
I think that dataset templates works a little bit differently, but I can be wrong. If you can quickly check by creating the template (withthout filling periodo field) and try to display it's details I would appreciate this. If everything is okay then I guess we can not include the check

Reply:
checked, no check needed. if the field value is empty or blank in the edit view, the field does not get created.